src/pages/NewProposal.tsx

```diff
@@ -58,7 +58,7 @@ export const useNewProposalFormContext = () =>
   useContext(NewProposalFormContext);
 
 export type NewProposalFormProviderProps = {
-  children: any;
+  children: any;w
```

Comment:
I don't think this 'w' is meant to be here.
Did you know VScode has an option to view what you've staged and what you're about to commit? I think github desktop must have something similar :).

Reply:
the w isn't supposed to be there, and the date time picker is temporary. shadcn's new version contains a date picker, so I was thinking on waiting on that one, this will be the MVP of the date picker